x-pack/plugins/uptime/public/components/monitor/status_details/status_bar/monitor_redirects.tsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { useState } from 'react';
+import { i18n } from '@kbn/i18n';
+import { EuiPopover } from '@elastic/eui';
+import styled from 'styled-components';
+import { Ping } from '../../../../../common/runtime_types';
+import { PingRedirects } from '../../ping_list/ping_redirects';
+import { MonListDescription, MonListTitle } from './status_bar';
+
+interface Props {
+  monitorStatus: Ping | null;
+}
+
+const RedirectBtn = styled.span`
+  cursor: pointer;
+`;
+
+export const MonitorRedirects: React.FC<Props> = ({ monitorStatus }) => {
+  const list = monitorStatus?.http?.response?.redirects;
+
+  const [isPopoverOpen, setIsPopoverOpen] = useState(false);
+
+  const button = (
+    <MonListDescription>
+      <RedirectBtn
```

Comment:
i think it was avoided because of the reason EuiLink doesn't support react router, but with my other PR, that util is added, so i will refactor this there.